src/nupic/frameworks/opf/htm_prediction_model.py

```diff
@@ -1335,8 +1344,8 @@ def read(cls, proto):
     model = cls(spEnable=spEnable,
                 tmEnable=tmEnable,
                 clEnable=clEnable,
-                inferenceType=inferenceType,
-                network=network)
+                network=network,
+                baseProto=proto.modelBase)
```

Comment:
The `network` arg appears to only be passed from this `read` method in the entire `nupic` body of code.
```
~/nta/nupic $ git grep 'network='

src/nupic/frameworks/opf/htm_prediction_model.py:      network=None,
src/nupic/frameworks/opf/htm_prediction_model.py:                network=network,
vkruglikov@Vitalys-MacBook-Pro:~/nta/nupic (nup2464-htmpredmodel-serialize-missing-fields)$ 

```